Community.md

```diff
@@ -0,0 +1,47 @@
+# Community Continuity Guidelines v2.0.0
+
+As the creators, and maintainers of this project. We want to ensure that the project lives and continues to grow. Not blocked by any singular person's computer time.
+
+One of the simplest ways of doing this, is by encouraging a larger set of shallow contributors.
+
+#### Ownership
+
+If you get a merged PR, regardless of content (typos, code, doc fixes) then you're eligible for push access to this organization. We check for this on PR merges and extend an invite via GitHub.
+
+Offhand, it's easy to imagine that this would make code quality suffer, but in reality it offers fresh perspectives to the codebase and encourages ownership from people who are depending on the project. If you are working with a codebase, then you probably have the skills to improve it, and offer valuable feedback.
+
+Everyone comes in with their own perspective on what a project could/should look like, and encouraging discussion can help expose good ideas sooner.
+
+#### Why do we give out push access?
+
+It can be overwhelming to suddenly be offered the chance to wipe the source code for a project. Don't worry, we don't let you push to master.
+
+All code is peer-reviewed, and we have the convention that someone other than the submitter should merge the pull request.
+
+As an organization contributor you can merge other people's pull requests, or other contributors can merge yours. You won't be assigned a pull request, but you're welcome to just jump in and take a code review on things that interest you.
+
+If it feels right, merge.
+
+This project is not continuously deployed, there is space for debate after review too. Offering the chance to revert, or make an amending pull request.
+
+It's convention to provide a call to action via `/cc @[org]/contributors` to invite discussion.
+
+#### How can we help you get comfortable contributing?
+
+It's normal for a first PR to be a potential fix for a problem, moving on from there to helping the project's direction can be difficult. We try to help contributors cross that barrier by offering good first step issues. These issues can be done without feeling like you're stepping on toes. Ideally, these are non-critical issues that are well defined. They will be purposely avoided by mature contributors to the project.
+
+We aim keep all project discussion inside GitHub issues. This is to make sure it is public. If you have questions about how to use the library, or how the project is running GitHub issues are the goto tool.
+
+#### Our expectations on you as a contributor
+
+To quote [@alloy](https://github.com/alloy) from [this issue](https://github.com/Moya/Moya/issues/135):
+
+> Don't ever feel bad for not contributing to open source.
+
+We want contributors to both provide ideas, keep the ship shipping and to take some of the load from others. It is non-obligatory; we're here to have fun. :trophy:
```

Comment:
Agreed I don't have a CONTRIBUTING.md file on my repo yet (that's one of the issues still open) so for now I just point to Moya's one until I find time to adapt to mine. But still, 2 of the last people I gave push access to, informing them in a comment in the PR, were quite "afraid" to receive that push access as in "no no no we won't have more time to spent on your project to help you maintain it, that was only a one-time PR", so I feel it might be interesting to add a paragraph here just to be extra-clear. Like:

> The fact that we will give you push access doesn't mean that we expect you to contribute more actively to the project or be more active. It's up to you to use that push access or don't use it at all, that's just a possibility we give you without any obligation whatsoever.

> But the fact that you'll have push access would allow you to (1) avoid having to fork the project if you want to submit other PRs as you'll be able to create branches directly on the project (2) help triage issues, merge PRs, but it's up to you to use those superpowers or not :wink:

Thoughts?